lang/perl-file-rsyncp/patches/no-subdirs.diff

```diff
@@ -0,0 +1,11 @@
+--- a/Makefile.PL	2010-07-25 22:50:02.000000000 +0200
```

Comment:
I'd like all of these .diff files renamed to .patch and prefixed with a number (eg 010, 020, etc...). This is done so that the order can easily be determined.

Reply:
OK, done